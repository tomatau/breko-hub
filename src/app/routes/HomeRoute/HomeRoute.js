import React from 'react'
import { Helmet as DocumentMeta } from 'react-helmet-async'
import { homeRoute as homeRouteCopy } from 'app/copy'

export default class HomeRoute extends React.Component {
  render() {
    return (
      <>
        <DocumentMeta>
          <title />
        </DocumentMeta>
        <section className='HomeRoute'>
          {homeRouteCopy.content}
        </section>
      </>
    )
  }
}
