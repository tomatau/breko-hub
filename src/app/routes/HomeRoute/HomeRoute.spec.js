import React from 'react'
import HomeRoute from './HomeRoute'

describe(`HomeRoute Component`, function () {
  helpers.setupSnapshots(__filename)

  it(`matches snapshot`, () => {
    expect(
      snap(shallow(<HomeRoute />))
    ).to.matchSnapshot()
  })
})
