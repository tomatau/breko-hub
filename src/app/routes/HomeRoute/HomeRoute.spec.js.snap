// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeRoute Component matches snapshot 1`] = `
<Fragment>
  <Helmet
    defer={false}
    encodeSpecialCharacters={true}
  >
    <title />
  </Helmet>
  <section
    className="HomeRoute"
  >
    <h2>
      Welcome to Breko-Hub!
    </h2>
    <p>
      A boilerplate designed for quick development!
    </p>
    <p>
      There are examples showing how you can customise to your needs.
    </p>
    <p>
      The server routes also auto update on changes!
    </p>
  </section>
</Fragment>
`;
